index.js

```diff
@@ -46,130 +55,36 @@ const createTables = async (pool) => {
             )),
             size VARCHAR(10) NOT NULL,
             type VARCHAR(80) NOT NULL,
-            stock_count INTEGER NOT NULL,
+            stock_count INT NOT NULL,
             price FLOAT NOT NULL 
         );
+        CREATE TABLE orders(
+            id SERIAL PRIMARY KEY,
+            date_placed DATE NOT NULL,
+            status VARCHAR NOT NULL CHECK(status IN(
+                'pending', 
+                'awaiting fulfillment', 
+                'awaiting shipment', 
+                'shipped',
+                'completed',
+                'cancelled',
+                'refunded'
+            )),
+            customer_id INT NOT NULL REFERENCES customers(id)
+        );
+        CREATE TABLE orders_costumes(
+            id SERIAL PRIMARY KEY,
+            order_id INT REFERENCES orders(id),
+            costume_id INT REFERENCES costumes(id)
+        );
     `)
 }
 
-const createCostume = async (
```

Comment:
heads up - for code changes that move code around, I recommend a separate PR for refactor and for the new features. It makes it easier to review code. It's not a big deal, just for future reference

Reply:
Noted!